urllib3/backends/__init__.py

```diff
@@ -1,6 +1,9 @@
-class LoopAbort(Exception):
-    pass
-
-from .trio_backend import TrioBackend
+from urllib3.packages import six
 from .sync_backend import SyncBackend
-from .twisted_backend import TwistedBackend
+
+__all__ = [SyncBackend]
```

Comment:
`__all__` is supposed to be a list of strings, not objects :-)

Reply:
Oops!